src/RMP/Translate/lang/programmes/en.po

```diff
@@ -1044,6 +1044,10 @@ msgstr "Last Played"
 msgid "latest"
 msgstr "Latest"
 
+#. English: "Latest Episode".
+msgid "latest_episode"
+msgstr "Latest Episode"
```

Comment:
The standard (see the rest of the file) is only the first word is capitalised. So this should probably be
```Latest episode```